ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -148,6 +149,19 @@ data:
       # We had to resort to a fixed list here because there are a lot of openshift-RAND namespaces created.
       # (must-gather, debug, amq, test, etc) If you want your namespaces to get it's own indexed namespace
       # label, add it here, in alphabetical order, and to the near identical line below with =~ instead of !~.
+      #
```

Comment:
It looks like you've separated the above comment from the config it refers to.

Reply:
removed